modules/special-rules/pages/walker.adoc

```diff
@@ -0,0 +1,22 @@
+= Walker
+
+Some vehicles with legs are particularly manoeuvrable.
+
+[NOTE.e40k]
+====
+We've brought this ability into the main body of the rules.
+In _Epic 40,000_ it was a suggestion in the _Battles Book_, later confirmed in Official Rule Amendments and Errata, _Firepower_, issue 1.
```

Comment:
What happened to the text for players who don't use the Firepower changes? That wasn't included randomly, see the [FB discussion](https://www.facebook.com/groups/1952676958177757/permalink/3751581358287299/).

Reply:
I thought it was redundant in the note, potentially even a little patronising. The note already says that it was a 'suggestion, later confirmed in Firepower' — so this fact is already highlighted, and people will pick and choose what they want without us telling them that they can. We can put it back in, but I really don't see the point — highlight the provenance/difference between E40k and ER, and move on.